## C0330 (bad-continuation)

### :x: Problematic code:

```python
def foo():
print('Hello')
pass
```

### :heavy_check_mark: Correct code:

```python
def foo():
  print('Hello')
  pass
```

### Rationale:

Improper indentation.

### Related resources:

- [Issue Tracker](https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22bad-continuation%22+OR+%22C0330%22)
